### ISSUE TITLE
Cannot create initial behaviour tree node

### ISSUE THREAD

**Issue by coolmule0** [bitbrain/beehave#202]
Godot version: 4.1
Beehave version 2.7.4

**Describe the bug**
Cannot create a beehave tree node. Editor throws an error in the output and nothing is changed in the scene.
Editor output:
```
Cannot get class ''.
editor/editor_data.cpp:581 - Parameter "p_object" is null.
editor/scene_tree_dock.cpp:2275 - Condition "!child" is true.
```

**To Reproduce**
1. Download and install beehave as instructed.
2. Create a new node anywhere (as a child, as a new scene) of type `BeehaveTree(beehave_tree.gd)`
3. Error thrown

**Expected behavior**

**Desktop:**
 - OS: Linux - Arch

**Additional context**

**Comment by Nemesis-AS** [comment]
I do not understand what could be causing the issue, but a workaround is to make a change in the `addons/beehave/nodes/beehave_tree.gd` script, then reverting the change and saving the script.

This could be a potential issue with the engine, as it does not seems to be registering the script as a `@tool` before the script is saved after changing. But other `@tool` scripts are being loaded properly so I'm not sure what's wrong here

**Comment by coolmule0** [comment]
Yes! solves my issue

**Comment by AndreiSchapov** [comment]
Should probably be reopened, since the bug is still there. Just happened to me. I don’t quite get why the Tree node should be a @tool. For checking whether its children are beehave nodes?

**Comment by tayete** [comment]
Same here with Windows 10. Tried in two different pcs, one with Godot 4.0 and another with 4.1.1.
Both showed this same error.

**Comment by bitbrain** [comment]
A good solution for this probably is to remove `@tool` everywhere as Beehave is not really meant to be run inside the editor itself.

**Comment by CoreyAlexander** [comment]
This happened to me when installing via the Godot Asset Library (version 2.7.6).
When installing manually from the Github via the Zip (Version 2.7.9), I was able to get it installed and working.
The only difference I know of is that I manually had to drag and drop the `script_templates` folder from the zip into my project. I did not see that folder when getting it from the asset library.

I'm using Windows10, Godot 4.1.1.stable.mono.
